model/flow/transaction.go

```diff
@@ -53,16 +57,26 @@ func NewTransactionBody() *TransactionBody {
 	return &TransactionBody{}
 }
 
+// Fingerprint returns the canonical, unique byte representation for the TransactionBody.
+// As RLP encoding logic for TransactionBody is over-ridden by EncodeRLP below, this is
+// equivalent to directly RLP encoding the TransactionBody.
+// This public function is retained primarily for backward compatibility.
```

Comment:
I am a bit confused about backward compatibility aspect. I don't see it being used, maybe we can just remove it?